### DISCUSSION TITLE
Supabase Org ID

### DISCUSSION THREAD

**Discussion by DaveLofy1** [supabase/supabase#30595] [Questions] [answered]
How do I find my Supabase Org ID?

**Comment by GaryAustin1** [comment, marked as answer]
Do you mean Org ID or instance/project ID?
![image](https://github.com/user-attachments/assets/3de4143b-6c3c-409f-b880-bac62ee97277)

As far as I know orgs only have names.

**Reply by DaveLofy1** [reply]
I was not sure. I had come to the same conclusion that this might be what is being requested. I am going with this as the answer.

**Reply by DaveLofy1** [reply]
Thanks for the quick reply.
